### DISCUSSION TITLE
How to synchronise two databases with KeeShare?

### DISCUSSION THREAD

**Discussion by volkerwysk** [keepassxreboot/keepassxc#6118] [Q&A] [answered]
I'm trying to synchronise my password database between my destop and my laptop with KeeShare. I've followed the instructions in the user guide. I've configured the KeeShare container to reside in my personal Nextcloud cloud storage. The actual databases are in non-shared hard-disk memory. 

I can set up my desktop database to export+import and synchronize using new container, which gets written to disk. 

But how do I use the same container on the laptop? When I set up synchronisation with the container, it gets overwritten and nothing else happens. No passwords are imported. 

When I export on the desktop and import on the laptop, it works. It's just synchronisation which doesn't work.

Regards,
Volker

**Comment by volkerwysk** [comment, marked as answer]
After I've changed something on the desktop side, the container file gets written und die laptop side notices that. The passwords are then imported by the laptop. The synchronisation appears to work, and also in the other direction.

It remains that the synchronisation doesn't work at the beginning. Or is it supposed to be like that? 

Either way, it should be mentioned in the user guide.

Cheers,
Volker

**Reply by droidmonkey** [reply]
KeeShare UX is being overhauled in 2.7.0, it is currently not obvious what will happen when you choose different options.
